lib/framework/view/ensemble_page.dart

```diff
@@ -0,0 +1,322 @@
+import 'package:ensemble/framework/scope.dart';
+import 'package:ensemble/framework/widget/widget.dart';
+import 'package:ensemble/page_model.dart';
+import 'package:ensemble/util/utils.dart';
+import 'package:ensemble/widget/helpers/controllers.dart';
+import 'package:ensemble_ts_interpreter/invokables/invokable.dart';
+import 'package:flutter/material.dart';
+
+import '../data_context.dart';
+
+class PageWidget extends StatefulWidget {
```

Comment:
Are there any differences from ensemble.Page or did you just copy this over? I think @vusters would be the best person to take a look at this.

Reply:
@vusters can you take a look? has been open for a while

Reply:
we are abandoning this review as stated in the ticket. This is not hte right solution. I'll keep the ticket with me